common/perception_utils/include/perception_utils/transform.hpp

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef PERCEPTION_UTILS__TRANSFORM_HPP_
+#define PERCEPTION_UTILS__TRANSFORM_HPP_
+
+#include "geometry_msgs/msg/transform.hpp"
+
+#include <tf2_ros/buffer.h>
+#include <tf2_ros/transform_listener.h>
+
+#ifdef ROS_DISTRO_GALACTIC
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#else
+#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
+#endif
+
+#include "boost/optional.hpp"
+
+#include <string>
+
+namespace
+{
```

Comment:
I think unnamed namespaces in a header file should be avoided.
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf21-dont-use-an-unnamed-anonymous-namespace-in-a-header

Reply:
Thank you for helpful review.
As same as other fixing point, I would like to fix this point to prevent bigger changing point.